Topic-2-CSS/Task 2/Css/aStyles.css

```diff
@@ -0,0 +1,13 @@
+header, footer, aside, nav{
+  background-color: red;
+}
+
+html *
+{
```

Comment:
Please keep consistency in the way you define your CSS rules, my recommendation is to follow the same code style in the `normalize.css` file